clients/CMakeLists.txt

```diff
@@ -135,13 +135,15 @@ add_custom_command( OUTPUT "${HIPBLAS_GENTEST}"
 
 add_custom_target( hipblas-common DEPENDS "${HIPBLAS_COMMON}" "${HIPBLAS_TEMPLATE}" "${HIPBLAS_SMOKE}" "${HIPBLAS_GENTEST}" )
 
-rocm_install(
-  FILES ${HIPBLAS_COMMON} ${HIPBLAS_TEMPLATE} ${HIPBLAS_SMOKE}
-  DESTINATION "${CMAKE_INSTALL_BINDIR}"
-  COMPONENT clients-common
-)
-rocm_install(
-  PROGRAMS ${HIPBLAS_GENTEST}
-  DESTINATION "${CMAKE_INSTALL_BINDIR}"
-  COMPONENT clients-common
-)
+if(BUILD_CLIENTS_BENCHMARKS)
+  rocm_install(
```

Comment:
This looks wrong, if guarding it should be if( BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS) etc.

Reply:
hipblas-test is not marked as depending on clients-common.

Reply:
Probably can fix it all up after samples is merged